interface/weno/admin.php

```diff
@@ -64,16 +84,72 @@
 
 <br><br>
 <?php
-if (file_exists('../../contrib/weno/pharmacyList.csv')) {
-    echo "<a href='import_pharmacies.php?csrf_token_form=" . attr_url(CsrfUtils::collectCsrfToken()) . "' class='btn btn-default'>" . xlt("Import Pharmacies Script") . "</a> <br>";
-}
+
+
 ?>
+    <h2>Pharmacy Import / Update </h2><br>
+    <form class="form-inline" id="addpharmacies">
+        <div class="form-group">
+            <label for="city" ><?php print xlt("City");?></label>
+            <input type="text" class="form-control" id="city">
+        </div>
+        <div class="form-group">
+            <label for="state"><?php print xlt("State");?></label>
+            <input type="text" class="form-control" id="state">
+        </div>
+        <div class="form-group" id="form_btn">
+            <button type="submit"  class="btn btn-primary" >Submit</button>
+        </div>
+        <div class="form-group" id="loader"></div>
 
+    </form>
+    <p>
+        <h4>Disclaimer:</h4>
+        This is public information maintained by HHS/CMS. We cannot be responsible for the data received.<br>
+        Utilize at your own risk. Always verify data received.
+    </p>
 </div>
 <script>
-$(function(){
-    $("#addtolistid_state").hide();
+
+let f = document.getElementById('addpharmacies');
+f.addEventListener('submit', importPharm);
+$('#loader').hide();
+
+function importPharm(e) {
+    top.restoreSession();
+    let city = document.getElementById("city").value;
+    let state = document.getElementById("state").value;
+    if (city.length === 0 || state.length ===0) {
+        alert("City and state must both be filled out");
+        e.preventDefault();
+        return false;
+    }
+     //user experience hide the submit button and show spinner
+    $('#loader').show();
+    $('#form_btn').hide();
+    e.preventDefault();
+    $.ajax({
+        type: 'GET',
+        dataType: 'JSON',
+        url: 'pharmacyHelper.php?csrf_token_form=' + <?php echo js_url(CsrfUtils::collectCsrfToken()); ?>,
```

Comment:
do not need to url encode this (this is done automatically by the jquery in this case):
```
url: 'pharmacyHelper.php?csrf_token_form=' + <?php echo js_escape(CsrfUtils::collectCsrfToken()); ?>,
```

Reply:
check